source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -99,111 +106,104 @@ deployment and your security needs.
+#.  Exit root:
 
-            sudo systemctl enable docker.service
+    .. code-block:: bash
 
-      #. Create (or edit if it exists) the file 
-         :file:`/etc/systemd/system/docker.service.d/51-runtime.conf` and include the following lines:
+       exit
 
-         .. code-block:: bash
+Configure and run Kubernetes
+****************************
 
-            [Service]
-            Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime runc"
+This section describes how to configure and run Kubernetes with:
 
-      #. Create (or edit if it exists) the file :file:`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and include the following lines:
+* Docker and runc
+* CRI-O and kata-runtime
 
-         .. code-block:: bash
+Configure and run Docker + runc
+===============================
 
-            [Service]
-            Environment="KUBELET_EXTRA_ARGS="
+#.  Enable the Docker service:
 
-      #. Restart services:
+    .. code-block:: bash
 
-         .. code-block:: bash
+       sudo systemctl enable docker.service
 
-            sudo systemctl daemon-reload
-            sudo systemctl restart docker
-            sudo systemctl restart kubelet
+#.  Create (or edit if it exists) the file 
+    :file:`/etc/systemd/system/docker.service.d/51-runtime.conf` and include the following lines:
 
-   or:
+    .. code-block:: bash
 
-   b) Run Kubernetes with CRI-O + your desired runtime. You can use multiple
-      runtimes with CRI-O, including *runc* and *kata-runtime*. To use
-      CRI-O + *kata-runtime*:
+       [Service]
+       Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime runc"
 
-      #. Enable crio.service:
+#.  Create (or edit if it exists) the file :file:`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` and include the following lines:
 
-         .. code-block:: bash
+    .. code-block:: bash
 
-            sudo systemctl enable crio.service
+       [Service]
+       Environment="KUBELET_EXTRA_ARGS="
 
-      #. Restart services:
+#.  Enter the commands:
 
-         .. code-block:: bash
+    .. code-block:: bash
 
-            sudo systemctl restart crio
-            sudo systemctl restart kubelet
+       sudo systemctl daemon-reload
+       sudo systemctl restart docker
+       sudo systemctl restart kubelet
 
-Run Kubernetes for the first time
-*********************************
+#.  Initialize the master control plane with the command:
 
-#. Prepare your system to run Kubernetes for the first time with the
-   following commands, either:
+    .. code-block:: bash
 
-   a) If you are running Kubernetes with Docker + runc:
+       sudo kubeadm init --pod-network-cidr 10.244.0.0/16 --ignore-preflight-errors=SystemVerification
 
-      .. code-block:: bash
 
-         sudo systemctl daemon-reload
-         sudo systemctl restart docker
-         sudo systemctl restart kubelet
+Configure and run CRI-O + kata-runtime
+======================================
 
-   or:
+#.  Enable the CRI-O service:
```

Comment:
I believe you need a command preceding this one: 

`sudo systemctl enable docker.service`

@bktan8 Please confirm.  I will test this again today too.

Reply:
Let's put this in the "Configure Kubernetes" section.